lib/bolt/outputter/human.rb

```diff
@@ -286,6 +286,19 @@ def print_plans(plans, modulepath)
                         "details and parameters for a specific plan.")
       end
 
+      def print_topics(topics)
+        print_message("Available topics are:")
+        print_message(topics.join("\n"))
+        print_message("\nUse `bolt guide <topic>` to view a specific guide.")
+      end
+
+      def print_guide(filepath, _topic)
+        content = File.read(filepath)
```

Comment:
Can we handle loading the file in the CLI so it's not duplicated across each outputter?

Reply:
Updated.